tools/vf-component-library/src/site/404.njk

```diff
@@ -11,7 +11,7 @@ permalink: 404.html
 <div><!-- empty --></div>
 <div>
   <h1 class="vf-intro__heading">{{ title }}</h1>
```

Comment:
shouldn't the elements of a `vf-intro` component be inside a parent that has the `vf-intro` class on it? ;)

Reply:
Yeah, this bit of code is quite old. I'm going to attempt a refactor of the landing page to better address some of the use cases we've seen .. less pontificating about the VF and making it more "How do I get the code and use it"

Reply:
Just realised this isn't the homepage! 😬 

There is a vf-intro parent element (code is collapsed in the diff). But this code is also still quite old and needs a "modernisation".

Reply:
ugh, I think this is the third time you've reminded me of code being collapsed in a diff 😭